rcsb/workflow/tests/testBcifWorkflow.py

```diff
@@ -94,6 +94,7 @@ def computeBcif(
         f"{inHash}",
     ]
     cmd = " ".join(options)
+    time.sleep(10)
```

Comment:
Can probably go lower:

```suggestion
    time.sleep(3)
```